### PR TITLE
Remove romana support

### PR DESCRIPTION
**Pull request by olemarkus**

### CODE REVIEW

File: docs/releases/1.19-NOTES.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 # Breaking changes
 
+* Removed support for Romana networking provider.
```
Comment (johngmyers) [inline docs/releases/1.19-NOTES.md:9]:
```suggestion
* Support for the Romana networking provider has been removed.
```

File: pkg/apis/kops/validation/validation.go [hunk 2]
```diff
@@ -367,6 +367,7 @@ func validateNetworking(c *kops.ClusterSpec, v *kops.NetworkingSpec, fldPath *fi
 	}
 
 	if v.Romana != nil {
+		allErrs = append(allErrs, field.Forbidden(fldPath.Child("romana"), "support for Romana has been removed"))
 		if optionTaken {
```
Comment (johngmyers) [inline pkg/apis/kops/validation/validation.go:371]: This if block can be removed

Comment (olemarkus) [reply pkg/apis/kops/validation/validation.go:371]: Can it? If I remove that block, kops creates my cluster.

Comment (johngmyers) [reply pkg/apis/kops/validation/validation.go:371]: A validation error should prevent creation. How is that happening?

Comment (olemarkus) [reply pkg/apis/kops/validation/validation.go:371]: Sorry, I thought you were referring to the outer if. Yeah, this one can technically be removed.

**Comment by rifelpet** [comment]
this looks good to me too. i'm curious if/when we'd want to remove the `RomanaNetworkingSpec` type from the API altogether. If we remove it now, kops will error saying the field isnt recognized. I suppose having an explicit validation error is clearer to the user, but it might be a good idea to come up with a plan to remove the type altogether.

**Comment by olemarkus** [comment]
We can remove the struct from the next API version, but I think we should keep the unversioned struct for as long as any versioned API has the struct so that people run into the validation errors.

**Comment by justinsb** [comment]
Agree with keeping the API fields, to explicitly mark it deprecated and help users.  Also makes it easier if we change our mind and re-add support!

**Pull request merged**